compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TileAndDecomposeAttention.cpp

```diff
@@ -478,6 +476,34 @@ IREE::LinalgExt::AttentionOp tileAttention(IREE::LinalgExt::AttentionOp attnOp,
   return tiledAttentionOp;
 }
 
+Value scaleQuery(Value querySlice, Value scale, RewriterBase &rewriter) {
+  ShapedType queryType = cast<ShapedType>(querySlice.getType());
+  Location loc = querySlice.getLoc();
+
+  // Create a fill op for scale.
+  SmallVector<OpFoldResult> queryDims =
```

Comment:
Just wondering why we need a fill op for scale? Can it not be an arith.constant? Is it because scale is a value and not an attribute?

Reply:
Another way this can be done is having a linalg.generic with only a outs() mulf and multiplying it. I have no preference, this was just easy and generates good output.

Reply:
Doesn't really matter to me, because TileAndDecompose is a codegen detail, but fill + generic does seem a little strange to me. The single generic with an elementwise multiply is probably more canonical, but not for this PR.

Reply:
Sounds good, works for me